terraform/global-resources/elasticsearch.tf

```diff
@@ -324,4 +337,10 @@ resource "aws_elasticsearch_domain" "test" {
   ]
 }
 TESTPOLICY
+
+  log_publishing_options {
```

Comment:
Isn't this a duplicate of lines 250-253?

Reply:
It's on a different es cluster: `audit_1` vs `test`